wordpress-mobile/WordPress-Android
# [JP Content Migration Flow] Add success card to "My Site"

Fixes #17478 

<img width="362" alt="Screen Shot 2022-11-23 at 1 00 20 AM" src="https://user-images.githubusercontent.com/14964993/203467002-419799f8-799a-47be-a476-05ab386f2384.png">
<img width="362" alt="Screen Shot 2022-11-23 at 1 00 04 AM" src="https://user-images.githubusercontent.com/14964993/203467007-7ad7595f-bd31-490f-a887-634d50c4d271.png">

To test:
1 - Install WordPress;
2 - Install Jetpack using the same variant (e.g. if installed WordPress version was `jalapenoDebug`, install Jetpack `jalapenoDebug`);
3 - The migration success card **SHOULD NOT** appear on "My Site" ("HOME" or "MENU");
4 - Set the value of `isMigrationCompleted` to `true` (`org/wordpress/android/ui/mysite/MySiteViewModel.kt:438`);
5 - Build and install Jetpack again (like step 2);
6 - The migration success card **SHOULD** appear on "My Site" (on "HOME" **and** "MENU");

Step 4 is needed because this flag is not being set anywhere yet.

## Regression Notes
1. Potential unintended areas of impact
Order of items in "My Site".

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and unit testing.

3. What automated tests I added (or what prevented me from doing so)
Updated `MySiteViewModelTest.kt` and added `AppStatusTest.kt`.

PR submission checklist:
